site/en/docs/privacy-sandbox/unified-origin-trial/index.md

```diff
@@ -0,0 +1,371 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox relevance and measurement unified origin trial'
```

Comment:
```suggestion
title: 'Relevance and measurement unified origin trial'
```

Strongly recommend dropping "Privacy Sandbox" as the reader is already in our docs.

![DCC landing page, a long navigation.](https://user-images.githubusercontent.com/2766281/188676075-b59939ee-139c-409a-b70a-573742f8780d.png)

Reply:
I don't have an opinion on this either way, but I remember @rowan-m had a reasoning behind this specific title. Can we change it?

Reply:
Yeah, this seems fair—doubling up "Privacy Sandbox" in the "Privacy Sandbox" section is a bit redundant. What I would like to retain (which we do have) is the full origin trial name in the article body, as in if I search for "Privacy Sandbox Relevance and Measurement" then this article should come up.